logger.go

```diff
@@ -121,9 +125,15 @@ func (sumoLogger *sumoLogger) handleBatchedLogs() {
     for {
       err := sumoLogger.sendLogs(logBatch)
       if err == nil {
+        retryInterval = initialRetryInterval
         break
       }
+      logrus.Info(fmt.Sprintf("Sleeping for %s before retry...", retryInterval.String()))
```

Comment:
add unit in the log? it's always good to include the unit whenever you are logging some variable.